zerver/lib/bugdown/__init__.py

```diff
@@ -302,6 +303,15 @@ def get_tweet_id(url):
         return None
     return tweet_id_match.group("tweetid")
 
+def render_tex(tex):
+    # type: (Text) -> Text
+    katex = Popen(['npm', 'run', '-s', 'katex'], stdin=PIPE, stdout=PIPE)
+    stdout = katex.communicate(input=tex)[0]
+    if katex.returncode == 0:
+        return stdout
+    else:
+        return None
+
```

Comment:
As far as I know `subprocess.check_call()` doesn't allow piping in content through `STDIN` (neither do `subprocess.check_output()`). That's why I followed [this SO answer's](http://stackoverflow.com/a/10103704) advice, and used `Popen`.

Maybe there's some way to do it using `check_call()` that I don't know?